packages/myst-cli/src/session/types.ts

```diff
@@ -13,8 +13,8 @@ export type ISession = {
   configFiles: string[];
   store: Store<RootState>;
   log: Logger;
+  reloadConfigs(): void;
```

Comment:
There is likely a better way to do this, the configs are still cached on TOC changes, which is pretty annoying. :)